exercises/high-scores/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+edition = "2018"
+name = "high-scores"
+version = "4.0.0"
```

Comment:
just confirming that I looked at this and it matches the corresponding problem-specifications version :+1: . I haven't done a full review yet, but if I don't get to it in time I trust other maintainers' judgment, so no need to wait for me.